src/ui/units/dash/containers/Header/Header.tsx

```diff
@@ -34,28 +42,34 @@ type Props = StateProps & DispatchProps & OwnProps;
 
 type State = {};
 
+const MobileTocHeader = ({onClick}: {onClick: () => void}) => {
+    return (
+        <div className={b('toc-toggle')} onClick={onClick}>
+            {i18n('dash.table-of-content.view', 'label_table-of-content')}
+            <Icon data={ChevronDown} />
+        </div>
+    );
+};
+
 class Header extends React.PureComponent<Props, State> {
     state: State = {};
 
-    toggleFullscreenModeMobile = () => {
-        this.props.toggleFullscreenMode({
-            location: this.props.location,
-            history: this.props.history,
-        });
-    };
-
     render() {
         if (!this.props.entry) {
             return null;
         }
 
+        const handleTocHeaderClick = () => this.props.toggleTableOfContent();
```

Comment:
Optional: there is no need to use extra function for one line function. Could you use 
`<MobileTocHeader onClick={this.props.toggleTableOfContent} />` here?